src/main/java/edu/iis/powp/adapter/PloterDrawerAdapter.java

```diff
@@ -31,7 +32,10 @@ public void drawTo(int x, int y)
     	line.setStartCoordinates(this.startX, this.startY);
         line.setEndCoordinates(x, y);
 
-		drawLine(line);
+		ApplicationWithDrawer.getDrawPanelController().drawLine(line);
```

Comment:
1. Constructor injection should be used (using a parameter). This is unnecessary direct coupling with ApplicationDrawer class.
2. Inheritance is not used (no super drawLine calls). Should be removed!